packages/react-environment/src/webb-context/bridge/bridge-api.ts

```diff
@@ -0,0 +1,74 @@
+import { ChainTypeId, chainTypeIdToInternalId, InternalChainId, WebbCurrencyId } from '@webb-dapp/apps/configs';
+import { BehaviorSubject, Observable } from 'rxjs';
+
+import { Currency } from '../currency/currency';
+
+export type BridgeCurrencyIndex = string | number | WebbCurrencyId;
```

Comment:
Could we narrow this type?

```suggestion
export type BridgeCurrencyIndex = string | WebbCurrencyId;
```